bolt-simple-taproot.md

```diff
@@ -764,43 +764,50 @@ any keys aggregated via musig2 also are assumed to use the `KeySort` routine
 #### To Local Outputs
 
 For the simple taproot commitment type, we use the same flow of revocation or
-delay, but instead left the revocation case into the taproot key-spend case.
+delay, while re-using the NUMS point to ensure that the internal key is always
+revealed. This ensures that the anchor outputs can _always_ be spent given
```

Comment:
i think it is worth specifying which internal key is being revealed. We want to reveal the internal key of the to_local anchor output.